### DISCUSSION TITLE
Add arcminutes and arcseconds for sexagesimal system

### DISCUSSION THREAD

**Discussion by lulunac27a** [cosinekitty/astronomy#341] [Ideas]
I want to see astronomy positions in [sexagesimal notation](https://en.wikipedia.org/wiki/Minute_and_second_of_arc) like arcminutes and arcseconds. An arcminute is 1/60th of degree and an arcsecond is 1/60th of arcminute.

**Comment by cosinekitty** [comment]
Hi @lulunac27a and welcome to the project!

People have asked for this as a feature before, but I have resisted it. The main reason is that Astronomy Engine is intended as a  calculation engine for astronomy, not a user interface or string formatting library. People want a lot of different options and features for how to format various angles, different precision rounding, HTML support, etc. A numeric formatter ends up being a culture-aware string processing library with significant complexity. The mathematics behind it is very simple: take fractional part, multiply by 60, etc. So my judgment is everyone can figure out their own chosen way of displaying numbers without me getting involved in disagreements about style, etc.
